packages/runtime/datastore/package.json

```diff
@@ -67,8 +67,8 @@
 		"@fluidframework/datastore-definitions": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/driver-utils": "workspace:~",
-		"@fluidframework/protocol-base": "^1.0.1",
-		"@fluidframework/protocol-definitions": "^1.1.0",
+		"@fluidframework/protocol-base": "^2.0.0",
```

Comment:
Another tangent to the PR: also see no imports from `@fluidframework/protocol-base` in this package, and in this case it's not just a dev dependency. Is this supposed to be here?

Reply:
Def not, should be deletable